corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -42,7 +42,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
 
                 for (let prop of group.properties) {
                     const isGeoCaseProp = (self.geoCaseProp === prop.name);
-                    var propObj = propertyListItem(prop.name, prop.label, false, prop.group, self.name, prop.data_type,
+                    var propObj = propertyListItem(prop.name, prop.label, false, prop.group_name, self.name, prop.data_type,
```

Comment:
Sorry for not being clear regarding the change needed here. `prop.group_name` would not work here since that is not sent through the `data_dictionary_json` view. `group.name` can be used instead of the `prop.group_name`.

```suggestion
                    var propObj = propertyListItem(prop.name, prop.label, false, group.name, self.name, prop.data_type,
```

Reply:
ok, got it.